VRDR/RaceAndEthnicity.cs

```diff
@@ -72,6 +74,29 @@ public static List<string> GetBooleanRaceCodes()
             booleanRaceCodes.Add(NvssRace.OtherRace);
             return booleanRaceCodes;
         }
+        /// <summary> GetDisplayValueForCode returns the display value for a race code, or the code itself if none exists</summary>
+        public static string GetDisplayValueForCode(string code) {
+            switch (code) {
+                case BlackOrAfricanAmerican:
+                    return "Black Or African American";
+                case AmericanIndianOrAlaskanNative:
+                    return "American Indian Or Alaskan Native";
+                case AsianIndian:
+                    return "Asian Indian";
+                case OtherAsian:
+                    return "Other Asian";
+                case NativeHawaiian:
+                    return "Native Hawaiian";
+                case GuamanianOrChamorro:
+                    return "Guamanian Or Chamorro";
+                case OtherPacificIslander:
+                    return "Other Pacific Islander";
+                case OtherRace:
+                    return "Other Race";
+                default:
+                    return code;
+            }
+        }
```

Comment:
Is this used anywhere?

Reply:
Yes in testing and appears in json output.